core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java

```diff
@@ -1109,6 +1109,7 @@ public <T> CompletionStage<T> invokeCommands(Collection<Address> targets,
                continue;
 
             ReplicableCommand command = commandGenerator.apply(target);
+            traceCommand(command, request);
```

Comment:
Is it okay to have multiple spans created that are not dependent to each other or does it show as concurrent? I worry it might show as each command will be dependent upon the next span and they can complete in different orders than submitted.

Reply:
All the spans will be correlated by the parent scope, e.g., the request invoked on the HotRod / REST client, that in turn can be correlated by the client side scope and so on and so forth...